task.sql

```diff
@@ -1,11 +1,25 @@
--- Use our database
-USE ShopDB; 
+USE ShopDB;
 
--- Some data should be created outside the transaction (here)
+INSERT INTO Products (ID, Name, Description, Price, WarehouseAmount)
+VALUES (1, 'AwersomeProduct', 'An awesome product!', 100, 50);
```

Comment:
There is a typo in the product name 'AwersomeProduct'. It should be 'AwesomeProduct'.